.gitlab-ci.yml

```diff
@@ -12,6 +12,7 @@ stages:
   - TEAMING-CHECK
   - TOPOLOGY-CHECK
   - OPENEBS-INSTALL-CHECK
+  - OPENEBS-INSTALL-CHECK
```

Comment:
Why are you adding stage here ? @uditgaurav

Reply:
Because it occurs due to conflicts